packages/consumption/src/modules/attributes/AttributesController.ts

```diff
@@ -1049,6 +1081,46 @@ export class AttributesController extends ConsumptionBaseController {
         }
     }
 
+    private async transmitDefault(attribute: LocalAttribute): Promise<void> {
+        if (!attribute.default) return;
+
+        const valueType = attribute.content.value.constructor.name as AttributeValues.Identity.TypeName;
+        const query: IIdentityAttributeQuery = {
+            valueType: valueType
+        };
+
+        const attributesWithSameValueType = await this.executeIdentityAttributeQuery(query);
+        if (attributesWithSameValueType.length <= 1) return;
+
+        const defaultCandidate = attributesWithSameValueType.find((attr) => !attr.succeededBy && attr.id.toString() !== attribute.id.toString());
+
+        if (!defaultCandidate) return;
```

Comment:
`defaultCandidate` should never be undefined, if we have valid data. So I am wondering, if we should throw an error here instead. However, if under any circumstances this case should be reached, the user couldn't delete their Attribute and also couldn't do anything about it.

Somehow I thought I already started this discussion, but that doesn't appear to be the case...

Reply:
From what I've seen here so far, I would expect an error without error code to let us know that our coding is faulty (would actually be like in the changeDefaultRepositoryAttribute error)

Reply:
If we throw an error, we would probably have to adjust the `query` above so that `defaultCandidate` really can't be `undefined`. See my other comment on this.